src/components/KRadio/KRadio.vue

```diff
@@ -1,7 +1,8 @@
 <template>
   <label
+    :checked="isSelected"
     class="k-radio"
-    :class="$attrs.class"
+    :class="`${isTypeDefault ? 'k-radio-default' : `k-radio-${type}`} ${$attrs.class ? $attrs.class : ''}`"
```

Comment:
suggestion: I think this logic can be simplified. 

Untested, so please make sure this looks correct
```suggestion
    :class="[isTypeDefault ? 'k-radio-default' : `k-radio-${type}`, ...$attrs.class]"
```

Reply:
Iterator won't work, but making the value an array rather than string does make it a little bit cleaner, thanks